changelog/Microsoft.IdentityProtectionServices.json

```
@@ -1,5 +1,31 @@
 {
 	"changelog": [
+		{
+			"ChangeList": [
+				{
+					"Id": "cad83874-f5f9-459e-9b42-965603408cda",
+					"ApiChange": "Property",
+					"ChangedApiName": "clientApplications",
+					"ChangeType": "Addition",
+					"Description": "Added the **clientApplications** property to the [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-beta) resource.",
```

Suggestion to enhance the description
```suggestion
					"Description": "Added the **clientApplications** property to the [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-beta) resource. Use this property to configure the service principals and workload identities to include in or exclude from a conditional access policy.",
```